src/libs/actions/PersonalDetails.js

```diff
@@ -135,6 +148,10 @@ function fetchPersonalDetails() {
             }
 
             const allPersonalDetails = formatPersonalDetails(personalDetailsList);
+            const hasConciergeDetails = _.some(allPersonalDetails, person => person.login === CONST.EMAIL.CONCIERGE);
+            if (!hasConciergeDetails) {
+                getPersonalDetailsForEmail(CONST.EMAIL.CONCIERGE);
```

Comment:
I know we have accepted a proposal already here... but we are currently refactoring our entire API and this solution is breaking one of our cardinal rules about not chaining API commands.

Not sure how you want to handle this @thienlnam, but I don't think this mergable.

It's a backend issue because if we are fetching personal details then the details of Concierge should be delivered  with the `personalDetailsList` and we should not have to make a second API call.